staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -56,29 +56,63 @@ var AllowedModes = []string{
 }
 
 const (
-	// Default configuration values for ModeBatch.
-	//
-	// TODO(ericchiang): Make these value configurable. Maybe through a
-	// kubeconfig extension?
-	defaultBatchBufferSize = 10000            // Buffer up to 10000 events before starting discarding.
-	defaultBatchMaxSize    = 400              // Only send up to 400 events at a time.
-	defaultBatchMaxWait    = 30 * time.Second // Send events at least twice a minute.
-	defaultInitialBackoff  = 10 * time.Second // Wait at least 10 seconds before retrying.
-
-	defaultBatchThrottleQPS   = 10 // Limit the send rate by 10 QPS.
-	defaultBatchThrottleBurst = 15 // Allow up to 15 QPS burst.
+	// Default configuration values.
+	DefaultBatchBufferSize = 10000            // Buffer up to 10000 events before starting discarding.
+	DefaultBatchMaxSize    = 400              // Only send up to 400 events at a time.
+	DefaultBatchMaxWait    = 30 * time.Second // Send events at least twice a minute.
+
+	DefaultBatchThrottleQPS   = 10 // Limit the send rate by 10 QPS.
+	DefaultBatchThrottleBurst = 15 // Allow up to 15 QPS burst.
+
+	DefaultInitialBackoff  = 10 * time.Second // Wait at least 10 seconds before retrying.
 )
 
 // The plugin name reported in error metrics.
 const pluginName = "webhook"
 
+// BatchBackendConfig represents batching webhook audit backend configuration.
+type BatchBackendConfig struct {
+	// BufferSize defines a size of the buffering queue.
+	BufferSize int
+	// MaxBatchSize defines maximum size of a batch.
+	MaxBatchSize int
+	// MaxBatchWait defines maximum amount of time to wait for MaxBatchSize
+	// events to be accumulated in the buffer before forcibly sending what's
+	// being accumulated.
+	MaxBatchWait time.Duration
+
+	// ThrottleQPS defines the allowed rate of batches per second sent to the webhook.
+	ThrottleQPS float32
+	// ThrottleQPS defines the maximum rate of batches per second sent to the webhook.
```

Comment:
ThrottleBurst defines ... webhook in case the capactiy ... was not utilized.

Reply:
Done, thanks